src/pydargs/utils.py

```diff
@@ -0,0 +1,20 @@
+from functools import partial
+from typing import Any, Callable, TypeVar
+
+Fct = TypeVar("Fct")
+
+
+def named_partial(func: Callable[..., Any], *, name: str, **kwargs) -> Callable[[str], Any]:
+    # Wrapper around partial to give it a name, for argparse to provide meaningful messages
+    result = partial(func, **kwargs)
+    setattr(result, "__name__", name)
+    return result
```

Comment:
```suggestion
def named_partial(func: Callable[..., Any], *, _display_name: str, **kwargs) -> Callable[[str], Any]:
    # Wrapper around partial to give it a name, for argparse to provide meaningful messages
    result = partial(func, **kwargs)
    setattr(result, "__name__", _display_name)
    return result
```

Reply:
Suggesting this to make the chance of collision smaller, e.g. if people add anything that actually expects an argument named `name`

Reply:
I think that probability is rather low, as that would have to be added in this package. But done 👍